docs/plugins/filters/oui.asciidoc

```diff
@@ -0,0 +1,71 @@
+:plugin: oui
+:type: filter
+:default_plugin: 0
+
+///////////////////////////////////////////
+START - GENERATED VARIABLES, DO NOT EDIT!
+///////////////////////////////////////////
+:version: v3.0.2
+:release_date: 2017-08-15
+:changelog_url: https://github.com/logstash-plugins/logstash-filter-oui/blob/v3.0.2/CHANGELOG.md
```

Comment:
Strange... The oui filter hasn't been updated recently (last on 2017-08-13), and yet it got picked up. Not sure what's going on with it. 

The oui-filter isn't included in the list of filters, and therefore doesn't appear in the Logstash Reference plugins. I could let this file go into the repo and still not link it.

Reply:
@elasticmachine run elasticsearch-ci/docs